### PR TITLE
Add description attribute to Teams AI Policy cmdlet documentation

### PR DESCRIPTION
**Pull request by Andy447**
Add description attribute to Teams AI Policy cmdlet documentation

### CODE REVIEW

File: teams/teams-ps/teams/New-CsTeamsAIPolicy.md [hunk 1]
```diff
@@ -82,6 +82,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Description
+Enables administrators to provide explanatory text about the Teams AI policy.
```
Comment (michbrown-png) [inline teams/teams-ps/teams/New-CsTeamsAIPolicy.md:86]: What are the possible values for this? Are there any limitations?

File: teams/teams-ps/teams/Set-CsTeamsAIPolicy.md [hunk 2]
```diff
@@ -121,6 +121,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Description
```
Comment (michbrown-png) [inline teams/teams-ps/teams/Set-CsTeamsAIPolicy.md:124]: What are the possible values for this? Are there any limitations?

**Review by michbrown-png — CHANGES_REQUESTED** [review]

File: teams/teams-ps/teams/New-CsTeamsAIPolicy.md (hunk 1 above)
Comment (Andy447) [reply teams/teams-ps/teams/New-CsTeamsAIPolicy.md:86]: @michbrown-png thanks for the review - bt:Description is the type that is used. This is used by almost all policies right now, and I borrowed the wording from their documentation - it should be any string

File: teams/teams-ps/teams/Set-CsTeamsAIPolicy.md (hunk 2 above)
Comment (Andy447) [reply teams/teams-ps/teams/Set-CsTeamsAIPolicy.md:124]: Should be any string for possible values - this is commonly used for all policies, you can search for "### -Description" in the document repo

**Pull request merged**